### PR TITLE
fix: vscode fails to load scripts from _solara/cdn

### PR DESCRIPTION
**Pull request by iisakkirotko**
Fixes https://github.com/widgetti/solara/issues/761.

### CODE REVIEW

**Comment by iisakkirotko** [comment]
* **#763** <a href="https://app.graphite.dev/github/pr/widgetti/solara/763?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @iisakkirotko and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: solara/components/markdown.py [hunk 1]
```diff
@@ -76,6 +76,12 @@ def _markdown_template(
     html,
     style="",
 ):
+    cdn = r"null"
```
Comment (maartenbreddels) [inline solara/components/markdown.py:79]:
```suggestion
    cdn = None
```

File: solara/components/markdown.py [hunk 2]
```diff
@@ -89,6 +95,10 @@ def _markdown_template(
 <script>
 module.exports = {
     async mounted() {
+        this.cdn = '"""
+        + cdn
+        + r"""';
+        console.log("cdn", this.cdn)
```
Comment (maartenbreddels) [inline solara/components/markdown.py:101]:
```suggestion
```

File: solara/components/markdown_editor.vue [hunk 3]
```diff
@@ -255,6 +255,7 @@ module.exports = {
       const labConfigData = document.getElementById('jupyter-config-data');
       if(labConfigData) {
         /* lab and Voila */
+        // console.log("HOHAODHWa", JSON.parse(labConfigData.textContent).baseUrl);
```
Comment (maartenbreddels) [inline solara/components/markdown_editor.vue:258]:
```suggestion
```

**Review by maartenbreddels — CHANGES_REQUESTED** [review]

File: solara/components/markdown.py (hunk 1 above)
Comment (iisakkirotko) [reply solara/components/markdown.py]: Let me know if you agree with my solution - I changed `cdn` to `None` by default, and instead filter it out later:
```python
        this.cdn = """
        + (rf"'{cdn}'" if cdn is not None else r"null")
        + r""";
```

Comment (maartenbreddels) [reply solara/components/markdown.py]: yeah, this makes more sense 👍

**Comment by maartenbreddels** [comment]
Related to https://github.com/widgetti/solara/pull/488

**Comment by maartenbreddels** [comment]
And https://github.com/widgetti/solara/pull/480

**Pull request merged**